yarn-project/aztec-node/src/sentinel/sentinel.ts

```diff
@@ -258,8 +263,13 @@ export class Sentinel extends (EventEmitter as new () => WatcherEmitter) impleme
    * We also don't move past the archiver last synced L2 slot, as we don't want to process data that is not yet available.
    * Last, we check the p2p is synced with the archiver, so it has pulled all attestations from it.
    */
-  protected async isReadyToProcess(currentSlot: bigint) {
-    const targetSlot = currentSlot - 2n;
+  protected async isReadyToProcess(currentSlot: SlotNumber): Promise<SlotNumber | false> {
+    if (currentSlot < 2) {
```

Comment:
Wait wait wait. We did `SlotNumber(foo)` a couple of files above. Is `SlotNumber = Number`?